Gemfile.lock

```diff
@@ -341,7 +341,7 @@ GEM
       thor (>= 0.18.1, < 2.0)
     rainbow (2.2.2)
       rake
-    rake (12.0.0)
+    rake (11.3.0)
```

Comment:
I've read through [changelog](https://github.com/ruby/rake/blob/master/History.rdoc#1200) of rake, I think it's no problem.